modules/_labs/core/react/stories/stories.tsx

```diff
@@ -21,7 +21,7 @@ export default {
 };
 
 export const Type = () => (
-  <section>
+  <section className="story">
```

Comment:
I don't think we need `className="story"`. I've not been including them and have been removing them when working on stories. If we want to add margins around stories, we can do that at the container level rather than adding the noise to stories.

Reply:
I also think that doesn't need to be done in this PR. I think we can have 1 PR that removes all these story wrappers.